conf/distro/include/omnect-os-distro.conf

```diff
@@ -160,3 +160,7 @@ OMNECT_DEVEL_TOOLS = "\
 "
 
 OMNECT_DEVEL_TOOLS:append:x86-64 = "acpica dmidecode"
+
+# make target architecture for visible for later processing
```

Comment:
-> "make target architecture ~~for~~ visible for later processing, e.g. in os-release"

Reply:
ack